src/actions.rs

```diff
@@ -33,9 +33,10 @@ pub fn list_episodes(search: &str) -> Result<()> {
                 .for_each(|(num, ep)| {
                     writeln!(
                         &mut handle,
-                        "({}) {}",
+                        "({}) {} [{}]",
```

Comment:
Changing the format is a breaking change, so I'd like this change behind configuration / a flag